bin/test-cli.sh

```diff
@@ -734,6 +759,23 @@ else
 	exit 1
 fi
 
+echo "Testing command ´sfcc-ci sandbox:update <sandbox> --ttl 2 --start-scheduler '{"weekdays":["MONDAY","TUESDAY","WEDNESDAY","THURSDAY","FRIDAY"],"time":"08:00:00+03:00"}' --stop-scheduler '{"weekdays":["MONDAY","TUESDAY","WEDNESDAY","THURSDAY","FRIDAY"],"time":"19:00:00Z"}'´:"
```

Comment:
Except updating the ttl should we cover changing the schedule?

Reply:
This is fixed,  I have removed the setting of ttl in all new tests added for schedule